docs/Services/URLOpening.md

```diff
@@ -0,0 +1,27 @@
+# URLOpening
+
+## Overview
+
+The `URLOpening` service provides shared functionality for opening URLs. Overriding this service can be helpful to determine when the SDK will attempt to open a given URL.
+
+## Declaration
+
+##### `public protocol URLOpening`
```

Comment:
Can you change this to `interface`

Reply:
Oh, the whole page should be removed. As we discussed, we don't need to show any details for the`UIService`.